Formula/gitsh.rb

```diff
@@ -7,6 +7,7 @@ class Gitsh < Formula
   env :std
   homepage 'https://github.com/thoughtbot/gitsh/'
   url 'https://thoughtbot.github.io/gitsh/gitsh-0.14.tar.gz'
+  head 'https://github.com/thoughtbot/gitsh.git'
```

Comment:
We need to add some additional dependencies for the `autogen` script to run. We can probably do something like this:

```ruby
head do
  url 'https://github.com/thoughtbot/gitsh.git'
  depends_on 'autoconf' => :build
  depends_on 'automake' => :build
end
```

I couldn't find docs for this `build { … }` style, but I see it in other formulae, e.g. the [GNU Octave formula][1]

[1]: https://github.com/Homebrew/homebrew-core/blob/master/Formula/octave.rb#L14-L22